src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/PublicAPI.Shipped.txt

```diff
@@ -333,8 +333,6 @@ Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedCollections.Re
 Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedCollections.RelatableItemBase.RelatableItemBase(string! name) -> void
 Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedCollections.RelationBase<TParent, TChild>
 Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedCollections.RelationBase<TParent, TChild>.RelationBase() -> void
-Microsoft.VisualStudio.ProjectSystem.VS.Tree.IFileIconProvider
-Microsoft.VisualStudio.ProjectSystem.VS.Tree.IFileIconProvider.GetFileExtensionImageMoniker(string! path) -> Microsoft.VisualStudio.Imaging.Interop.ImageMoniker
```

Comment:
Hmm. I hadn't realized there were public types that we might need to move. This would be a breaking change for anyone depending on the `IFileIconProvider` interface.

There are a couple of possibilities:
1. If we can prove that no one else uses or implements this interface we can just move it down to Managed.csproj as you have done here. Proving that might be difficult.
2. If not, we can move it and add a [`TypeForwardedTo` attribute](https://learn.microsoft.com/en-us/dotnet/standard/assembly/type-forwarding) to tell the runtime where to find the new version of the type. That requires that we keep the name of the type the same, including the namespace.

Reply:
@drewnoakes Any suggestions for where we might look to find implementations or uses of `IFileIconProvider` outside of our repo?

We could also play it safe and just assume we need to go with the `TypeForwardedTo` attribute.

Reply:
@tmeschter  I don't know if we can trust on Index, but it shows that no one else is using `IFileIconProvider`

Reply:
Added `TypeForwardedTo` attribute